pkgs/tools/security/tracee/default.nix

```
@@ -0,0 +1,55 @@
+{ stdenvNoCC, lib, fetchzip, autoPatchelfHook, elfutils, zlib }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tracee";
+  version = "0.6.5";
+
+  src = fetchzip {
+    url = "https://github.com/aquasecurity/tracee/releases/download/v${version}/tracee.tar.gz";
+    sha256 = "sha256-VgDJp+NsAV32kwa/GTrLnVH0IaTclsT9aD77SmMxVNM=";
+    stripRoot = false;
+  };
+
+  strictDeps = true;
+  nativeBuildInputs = [ autoPatchelfHook ];
+  buildInputs = [ elfutils zlib ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share/tracee}
+    mv dist/tracee-{ebpf,rules} $out/bin
+    mv dist/rules $out/share/tracee/
+
+    runHook postInstall
+  '';
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+
+    $out/bin/tracee-ebpf --help
+    $out/bin/tracee-ebpf --version | grep "v${version}"
+
+    $out/bin/tracee-rules --help
+
+    runHook postInstallCheck
+  '';
+
+  meta = with lib; {
+    homepage = "https://aquasecurity.github.io/tracee";
```

That site doesn't exist.
```suggestion
    homepage = "https://aquasecurity.github.io/tracee/latest";
```
This one does.

---

I was trying that link out but yeah it didnt work. I didnt mean to commit that one